### PR TITLE
fix #31764 gopokemongo.ru

### LINKED ISSUE #31764: gopokemongo.ru
### Issue URL (Annoyance)
[https://gopokemongo.ru/sajt-perehodit-na-protokol-https-vozmozhny-problemy-v-rabote-sajta.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgopokemongo.ru%2Fsajt-perehodit-na-protokol-https-vozmozhny-problemy-v-rabote-sajta.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 6.3.1399.4073
Browser: | Vivaldi, Opera, Яндекс
AdGuard driver: | TDI
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31764

### CODE REVIEW

File: RussianFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -887,9 +887,9 @@ fclmnews.ru#$#.col_02 > div.top_3 { position: absolute!important; left: -3000px!
 ! https://forum.adguard.com/index.php?threads/10880/
 sinoptik.ua#$#iframe[bb] { position: absolute!important; left: -3000px!important; }
 sinoptik.com.ru#$#iframe[bb] { visibility: hidden!important; }
-! https://github.com/AdguardTeam/AdguardFilters/issues/4368
-arendator.ru#$#.arcticmodal-container { display: none!important; }
-arendator.ru#$#.arcticmodal-overlay { display: none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/4368 https://github.com/AdguardTeam/AdguardFilters/issues/31764
```
Comment (BlazDT) [inline RussianFilter/sections/general_extensions.txt]: Please put new issue into a new line with comment

File: RussianFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -887,9 +887,9 @@ fclmnews.ru#$#.col_02 > div.top_3 { position: absolute!important; left: -3000px!
...
+gopokemongo.ru,arendator.ru#$#.arcticmodal-container { display: none!important; }
+gopokemongo.ru,arendator.ru#$#.arcticmodal-overlay { display: none!important; }
 arendator.ru#$#body { overflow: visible!important; }
```
Comment (BlazDT) [inline RussianFilter/sections/general_extensions.txt]: Please add domain also to this element or else scrolling is broken.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/general_extensions.txt (hunk 2 above)
Comment (zloyden) [reply RussianFilter/sections/general_extensions.txt]: I did not observe this in myself. Also checked with/without `gopokemongo.ru#$#body { overflow: visible!important; }` before commit, but i add this now. Thanks!

**Review by BlazDT — APPROVED** [review]

File: AnnoyancesFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -96,6 +96,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 eva.ru##div:has(> div > .subscribe-form)
 finanzen.net##.teaser > .box:has(> .box-headline:contains(WhatsApp Newsletter))
 finanzen.net##.col-sm-4 > .box:has(> .box-headline:contains(WhatsApp Newsletter))
+gopokemongo.ru#?##secondary > .sidebar-widget:has(> .textwidget > div > div[class$="form_line"])
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/css_extended.txt]: `?` is redundand. Please check article about our Ext CSS syntax.

File: RussianFilter/sections/general_extensions.txt [hunk 4]
```diff
@@ -887,10 +887,11 @@ fclmnews.ru#$#.col_02 > div.top_3 { position: absolute!important; left: -3000px!
 ! https://forum.adguard.com/index.php?threads/10880/
 sinoptik.ua#$#iframe[bb] { position: absolute!important; left: -3000px!important; }
 sinoptik.com.ru#$#iframe[bb] { visibility: hidden!important; }
-! https://github.com/AdguardTeam/AdguardFilters/issues/4368
-arendator.ru#$#.arcticmodal-container { display: none!important; }
-arendator.ru#$#.arcticmodal-overlay { display: none!important; }
-arendator.ru#$#body { overflow: visible!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/4368
```
Comment (Alex-302) [inline RussianFilter/sections/general_extensions.txt]: remove another report URL. It is redundant

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**